### ISSUE TITLE
Set the K by user

### ISSUE THREAD

**Issue by Jack-Lin-DS-AI** [GGiecold/Cluster_Ensembles#7]
Currently, there is an option to define the maximum K of the cluster number (N_clusters_max). Is it possible to define K by user instead of estimating K by the algorithm? Thank you.

**Comment by GGiecold-zz** [comment]
Starting with a large value of K, run Cluster_Ensembles. If satisfied with
the partition thereby obtained, stop. Otherwise, set K to a smaller value
and repeat the procedure until a partition with a desired number of
consensus clusters has been identified.

Gregory

On Oct 26, 2016 7:12 PM, "Chih-Hsu Lin" notifications@github.com wrote:

Currently, there is an option to define the maximum K of the cluster number
(N_clusters_max). Is it possible to define K by user instead of estimating
K by the algorithm? Thank you.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
https://github.com/GGiecold/Cluster_Ensembles/issues/7, or mute the thread

**Comment by Jack-Lin-DS-AI** [comment]
Let me rephrase my question. I'd like to get and compare the the clustering results of two different Ks, e.g., K=2 and K=3. Is it possible?
